daisy/workflow/workflow.go

```diff
@@ -225,9 +229,17 @@ func (w *Workflow) populateStep(step *Step) error {
 	}
 
 	// Recurse on subworkflows.
-	if step.SubWorkflow == nil {
-		return nil
+	if step.SubWorkflow != nil {
```

Comment:
These checks for step.SubWorkflow and step.IncludeWorkflow kinda make me think that Step needs a populate() method.